v3/plugins/ms-vscode/protobuf/0.4.2/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: ms-vscode
+name: protobuf
+version: 0.4.2
+type: VS Code extension
+displayName: Protobuf
+title: Google's protobuf .proto file support
+description: This extension adds rich language support for the proto3 syntax
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/zxh0/vscode-proto3
+category: Language
+firstPublicationDate: '2016-08-25'
```

Comment:
date should be fixed

Reply:
![image](https://user-images.githubusercontent.com/407397/91318685-e109a000-e791-11ea-8003-7e8106c3c5cb.png)
am I taking the wrong source for this field?

Reply:
I believe that date should match the date (roughly) the extension was published in our plugin registry (Che), not the extension's publish date.